devkit/samples/empty/empty.feature.ndjson

```diff
@@ -0,0 +1,11 @@
+{"meta":{"ci":{"buildNumber":"154666429","git":{"remote":"https://github.com/cucumber-ltd/shouty.rb.git","revision":"99684bcacf01d95875834d87903dcb072306c9ad"},"name":"GitHub Actions","url":"https://github.com/cucumber-ltd/shouty.rb/actions/runs/154666429"},"cpu":{"name":"x64"},"implementation":{"name":"fake-cucumber","version":"16.4.0"},"os":{"name":"linux","version":"6.8.0-36-generic"},"protocolVersion":"25.0.1","runtime":{"name":"node.js","version":"14.21.3"}}}
+{"source":{"data":"Feature: Empty Scenarios\n  Sometimes we want to quickly jot down a new scenario without specifying any actual steps\n  for what should be executed.\n\n  In this instance we want to stipulate what should / shouldn't run and what the output is\n\n  Scenario: Blank Scenario\n","mediaType":"text/x.cucumber.gherkin+plain","uri":"samples/empty/empty.feature"}}
+{"gherkinDocument":{"comments":[],"feature":{"children":[{"scenario":{"description":"","examples":[],"id":"0","keyword":"Scenario","location":{"column":3,"line":7},"name":"Blank Scenario","steps":[],"tags":[]}}],"description":"  Sometimes we want to quickly jot down a new scenario without specifying any actual steps\n  for what should be executed.\n\n  In this instance we want to stipulate what should / shouldn't run and what the output is","keyword":"Feature","language":"en","location":{"column":1,"line":1},"name":"Empty Scenarios","tags":[]},"uri":"samples/empty/empty.feature"}}
+{"pickle":{"astNodeIds":["0"],"id":"1","language":"en","name":"Blank Scenario","steps":[],"tags":[],"uri":"samples/empty/empty.feature"}}
+{"testRunStarted":{"timestamp":{"nanos":0,"seconds":0}}}
+{"testCase":{"id":"3","pickleId":"1","testSteps":[{"id":"2"}]}}
+{"testCaseStarted":{"attempt":0,"id":"4","testCaseId":"3","timestamp":{"nanos":1000000,"seconds":0}}}
+{"testStepStarted":{"testCaseStartedId":"4","testStepId":"2","timestamp":{"nanos":2000000,"seconds":0}}}
```

Comment:
So I think Line8/9 in the message are wrong. The rest look ok.

It's thinking there's a single undefined test step. Not sure where the bug lies here. But for now this is definitely something to fix in fake-cucumber